simulator/src/main/java/com/hazelcast/simulator/worker/WorkerCommandRequestProcessor.java

```diff
@@ -100,10 +100,9 @@ public void run() {
             while (running) {
                 try {
                     CommandRequest request = requestQueue.take();
-                    if (request == null) {
-                        throw new NullPointerException("request can't be null");
+                    if (request != null) {
```

Comment:
The contract of `BlockingQueue` forbids `null` values, the `offer` already fails. Just as precaution I left the check, but got rid of our own NPE.